### PR TITLE
[FL-1946] App launch

### PR DESCRIPTION
**Pull request by albkharisov**

### CODE REVIEW

**Review by DrZlo13 — APPROVED** [review]

File: application.proto [hunk 1]
```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package PB_App;
+option java_package = "com.flipperdevices.protobuf.app";
+
+message Start {
+    string name = 1;
+    string args = 2;
```
Comment (anna-oake) [inline application.proto:8]: String array?

File: application.proto [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+message LockStatusRequest {
```
Comment (LionZXY) [inline application.proto:11]: How I can unlock device? Maybe we can add flag in `Start` message?

Comment (albkharisov) [reply application.proto:11]: No ways to unlock, except for manually closing application.
This lock indicates that any other app is currently running.

File: application.proto (hunk 1 above)
Comment (albkharisov) [reply application.proto:8]: not array. 1 string for whole set of parameters

**Pull request merged**